content/docs/intro/v2_v3_migration.md

```diff
@@ -72,4 +72,7 @@ The migration use cases are as follows:
      2. Migrate Helm v2 configuration
      3. Migrate Helm v2 releases
      4. When happy that Helm v3 is managing all Helm v2 data (for all clusters and Tiller instances of the Helm v2 client instance) as expected, then clean up Helm v2 data
-   - The migration process is automated by the Helm [2to3](https://github.com/helm/helm-2to3) plugin
+
+3. Migration tools:
```

Comment:
## Reference

- [Blog explaining `2to3` plugin usage with examples](https://helm.sh/blog/migrate-from-helm-v2-to-helm-v3/) 
- [2to3](https://github.com/helm/helm-2to3) plugin

Reply:
done